src/components/ha-button-menu.ts

```diff
@@ -50,6 +51,46 @@ export class HaButtonMenu extends LitElement {
     `;
   }
 
+  protected firstUpdated(changedProps): void {
+    super.firstUpdated(changedProps);
+
+    this.updateComplete.then(() => {
```

Comment:
Can we do this only when in `rtl` mode?

Reply:
Done (getting hass with selector - equal to CSS host-context so no big deal).